From 5964c244b63d87f775e666157d1470ad91eb8e40 Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Fri, 15 Dec 2017 17:28:56 +0100
Subject: [PATCH 2/2] Take the standard $CXXFLAGS into account.

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 263d070..305999c 100644
--- a/Makefile
+++ b/Makefile
@@ -60,7 +60,7 @@ endif
 
 OPTFLAGS = -O2 -std=c++11 -fno-rtti
 
-CCFLAGS = -Wall $(OPTFLAGS) $(OPT_ENABLE_DEBUGGER_SUPPORT)
+CCFLAGS = -Wall $(CXXFLAGS) $(OPTFLAGS) $(OPT_ENABLE_DEBUGGER_SUPPORT)
 
 ifdef V8_SRCDIR
 override CPPFLAGS += -I$(V8_SRCDIR) -I$(V8_SRCDIR)/include
-- 
2.14.3

diff --git a/Makefile b/Makefile
index f410301..2ed035b 100644
--- a/Makefile
+++ b/Makefile
@@ -43,7 +43,7 @@ ifndef DISABLE_DIALECT
 REGRESS += dialect
 endif
 
-SHLIB_LINK += -lv8
+SHLIB_LINK += -lv8 $(RPM_HACK_LDFLAGS)
 ifdef V8_OUTDIR
 SHLIB_LINK += -L$(V8_OUTDIR)
 else
